components/global/user_guide_dropdown/user_guide_dropdown.tsx

```diff
@@ -19,6 +21,8 @@ type Props = {
     helpLink: string;
     reportAProblemLink: string;
     enableAskCommunityLink: string;
+    showGettingStarted: boolean;
+    showNextStepsTips: boolean;
```

Comment:
same here

Reply:
I am not sure we need this here as the initial display is handled by channel_view and the menu item just serves as unhiding it. (see previous functionality)